tests/benchmark_dht.py

```diff
@@ -2,7 +2,6 @@
 import argparse
 import random
 from warnings import warn
-import dht
 from tqdm import trange
 import dht
```

Comment:
Is this to fix the extra `import dht` or did you mean to remove both?

Reply:
yep, we still need L7 for dht.DHT at L22